specification.md

```diff
@@ -1631,6 +1631,168 @@ ThrottleSubmitter().start()
 ```
 
 
+#### Submit a malformed or unsatisfiable job
+
+Use the exception types to distinguish between re-triable or not
+
+```python
+import jpsi
+
+jex = jpsi.JobExector()
+job_1 = jpsi.Job()
+job_2 = jpsi.Job()
+
+spec_1 = jpsi.JobSpec()
+spec_2 = jpsi.JobSpec()
+
+spec_1.executable = '/bin/true'
+spec_2.executable = True   # type error
+
+job_1.specification = spec_1
+job_2.specification = spec_2
+
+try:
+    jex.submit(job_1)
+except jpsi.InvalidJobException:
+    # this should not happen
+    assert(True)
```

Comment:
Should this `assert(False)` to throw an assertion error in the exceptional case that this does occur?

Reply:
of course!